bot/exts/evergreen/status_cats.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from http import HTTPStatus
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class StatusCats(commands.Cog):
+    """Commands that give statuses described and visualized by cats."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=['statuscat'])
+    async def http_cat(self, ctx: commands.Context, code: int) -> None:
+        """Sends an embed with an image of a cat, potraying the status code."""
+        embed = discord.Embed(title=f'**Status: {code}**')
+        embed.set_image(url=f'https://http.cat/{code}.jpg')
+
+        try:
+            HTTPStatus(code)
+
+        except ValueError:
+            embed.set_footer(text='Inputted status code does not exist.')
```

Comment:
An image is set even if the status code doesn't pass the check in the following lines. This causes a funny side effect if you try `.statuscat 0`: [https://http.cat/0.jpg](https://http.cat/0.jpg) responds with a cat that's not even the 404 one, and it's displayed with the error message from the footer. You could move the `.set_image()` to the `try` or put it in an `else` block.
```suggestion


        try:
            HTTPStatus(code)
            embed.set_image(url=f'https://http.cat/{code}.jpg')

        except ValueError:
            embed.set_footer(text='Inputted status code does not exist.')
```

Reply:
Looks good to me!